### PR TITLE
21985 creationDate in isoformat

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#21985

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/tests/unit/resources/v2/test_reviews.py [hunk 1]
```diff
@@ -44,52 +44,49 @@ def basic_test_review():
...
-    review_three.save()
+    no_of_reviews = 11
+    create_test_review(no_of_reviews)
 
     rv = client.get(f'/api/v2/admin/reviews',
                     headers=create_header(jwt, [STAFF_ROLE], 'user'))
 
     assert rv.status_code == HTTPStatus.OK
-    assert 'reviews' in rv.json
+    assert len(rv.json.get('reviews')) == 10
```
Comment (severinbeauvais) [inline legal-api/tests/unit/resources/v2/test_reviews.py:80]: Shouldn't this be 11 as on line 73?

**Review by severinbeauvais — APPROVED** [review]

File: legal-api/tests/unit/resources/v2/test_reviews.py (hunk 1 above)
Comment (vysakh-menon-aot) [reply legal-api/tests/unit/resources/v2/test_reviews.py:80]: nope. its paginated to 10

Comment (vysakh-menon-aot) [reply legal-api/tests/unit/resources/v2/test_reviews.py:80]: total is 11

File: legal-api/tests/unit/resources/v2/test_reviews.py [hunk 2]
```diff
@@ -44,52 +44,49 @@ def basic_test_review():
...
+        review = Review()
+        review.filing_id = filing.id
+        review.nr_number = filing_dict['filing']['continuationIn']['nameRequest']['nrNumber']
+        review.identifier = filing_dict['filing']['continuationIn']['foreignJurisdiction']['identifier']
+        review.completing_party = 'completing party'
+        review.status = ReviewStatus.AWAITING_REVIEW
+        review.save()
+        reviews.append(review)
+
+    return reviews
```
Comment (ketaki-deodhar) [inline legal-api/tests/unit/resources/v2/test_reviews.py:61]: New learning for me here. Thanks Vysakh!

**Pull request merged**